### PR TITLE
Add support for Django 3.1

### LINKED ISSUE #31: Add support for Django 3.1
Django 3.1 RC1 has been released. In advance of the release itself, it would be good to be able to test with the RC1.

I'll cut a branch for it and submit a PR.

### PR DESCRIPTION
**Pull request by hugorodgerbrown**
Fixes #31 

~~Django 3.1 is still pre-release, which means I've had to pin the version in `tox.ini`, but I wanted to get a PR up and running to confirm that tests run.~~

Django 3.1 has now been released.

There is an open question re. the version modifier in `setup.py`, and whether it should be pinned at all - @jdufresne has an open PR (#30) to extend the test suite to support Django master - I think perhaps a change in `setup.py` should come with that PR - as it's more about the broader question of how far ahead the package should look in terms of Django version support. This PR is really just about getting 3.1 through the gate. (And both PRs are effectively non-functional, so can hopefully both be approved relatively easily.)

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -99,10 +99,11 @@ deps =
   coverage
   django22: Django>=2.2,<3.0
   django30: Django>=3.0,<3.1
+  django31: Django==3.1rc1
```
Comment (hugorodgerbrown) [inline tox.ini:102]: TODO: I don't know how to get tox to install a pre-release version, so have pinned.

Comment (jdufresne) [reply tox.ini:102]: Django 3.1 has now been released, so this can be changed to:

```suggestion
  django31: Django>=3.1,<3.2
```

File: setup.py [hunk 2]
```diff
@@ -32,5 +33,5 @@
         "Topic :: Utilities",
     ],
     python_requires=">=3.5",
-    install_requires=["Django>=2.2,<3.1", "requests"],
+    install_requires=["Django>=2.2,<3.2", "requests"],
```
Comment (jdufresne) [inline setup.py:36]: IMO, the upper bound should be unpinned to allow testing against Django master, see #30

Comment (hugorodgerbrown) [reply setup.py:36]: I think you're right, but the change should probably go through your PR #30 ?

Comment (jdufresne) [reply setup.py:36]: That approach works for me. Thanks.

**Comment by hugorodgerbrown** [comment]
@ubernostrum is there any chance of a merge / release for this and #30 ?

**Pull request closed without merging**